obsplus/events/utils.py

```diff
@@ -444,7 +444,8 @@ def get_seed_id(obj: catalog_component) -> str:
             except (TypeError, AttributeError):
                 raise AttributeError(f"Unable to fetch a seed id for {obj.resource_id}")
     # If it makes it this far, it could not find a non-None attribute
-    raise AttributeError(f"Unable to fetch a seed id for {obj.resource_id}")
+    # raise assertion error so this still works in validators
+    assert 0, f"Unable to fetch a seed id for {obj.resource_id}"
```

Comment:
I'm up for discussion on this, but I would actually rather this didn't raise an `AssertionError`. This isn't really meant for validating an object as much as it is a quick way for retrieving something stored on an object. I'm also not a big fan of using assert statements outside of testing.

Reply:
Changing this may also break code that uses this function.